modules/querying/pages/query-operations.adoc

```diff
@@ -1142,3 +1142,101 @@ The scope of the command depends on the user's current scope.
 If the user has set a working graph, then `DROP ALL` removes all the queries for that graph.
 If a user has set their scope to be global, then `DROP ALL` removes all queries across all graph spaces.
 
+== Query Descriptions
+
+Query descriptions explains the procedures for updating, showing, and dropping descriptions of queries and query parameters.
+
+=== Update description
+After creating a query, you can add a description for the query.
+
+*Syntax*
+[source]
+----
+UPDATE DESCRIPTION OF <objectType> <objectName> [ON GRAPH <graphName>] <description>
+----
+
+Required privilege
+`WRITE_QUERY`
+
+To update a description, run the `UPDATE DESCRIPTION` command as shown below. You can update the description of a query or a query parameter. If you choose not to specify a graph in the command, the current scope will be used as the scope of the description.
+
+[source, gsql]
+----
+GSQL > USE GRAPH POC_GRAPH
+GSQL > UPDATE DESCRIPTION OF QUERY query1 "xxx"
+GSQL > UPDATE DESCRIPTION OF QUERY query2 ON GRAPH graph1 "xxx"
+GSQL > UPDATE DESCRIPTION OF QUERY_PARAM query1.param1 "xxx"
+----
+
+[NOTE]
+====
+The term `UPDATE` here means adding a description to the target.
+If the target does not already have a description, it will be added.
+If the target already has a description, it will be replaced with the new one.
+====
+
+[CAUTION]
+====
+Using create or replace is preferable for updating a query to ensure that the description is retained.
+Otherwise, there is a risk of losing the description.
+====
+
+=== Show description
+
+*Syntax*
+[source]
+----
+SHOW DESCRIPTION OF <objectType> <objectName> [ON GRAPH <graphName>]
+----
+
+Required privilege
+`READ_QUERY`
+
+To display a description, use the `SHOW DESCRIPTION` command as shown below. Similar to the update command, you can display the description of a query or a query parameter.
+
+
+[source, gsql]
+----
+GSQL > USE GRAPH POC_GRAPH
+GSQL > SHOW DESCRIPTION OF QUERY query1
+GSQL > SHOW DESCRIPTION OF QUERY query1,query2
+GSQL > SHOW DESCRIPTION OF QUERY * ON GRAPH graph1
+GSQL > SHOW DESCRIPTION OF QUERY_PARAM query1.param1
+GSQL > SHOW DESCRIPTION OF QUERY_PARAM query1.param1,query1.param2
+GSQL > SHOW DESCRIPTION OF QUERY_PARAM query2.*
+----
+
+Support is provided for displaying descriptions of multiple targets simultaneously. You can separate the targets using commas, or use the wildcard `*` to display all descriptions.
+
+=== Drop description
+
+*Syntax*
+[source]
+----
+DROP DESCRIPTION OF <objectType> <objectName> [ON GRAPH <graphName>]
+----
+
+Required privilege
+`WRITE_QUERY`
+
+To drop a description, run the `DROP DESCRIPTION` command like below.
+Similar to the update command, you can drop the description of a global type object or a local type object.
+
+To remove a description, use the DROP DESCRIPTION command as shown below.
+Similar to the update command, you can remove the description of a query or a query parameter.
```

Comment:
This seems repetitive/redundant.  Is the second paragraph trying to say something new that the first paragraph didn't already say?

Tip: In regular English, the base meaning of "drop" is to fall, but in databases it has a particular technical interpretation.  Whereas "remove" means to take away. We want to use very straightforward, non-metaphorical English in our explanations. Therefore, I would use "remove" to explain what `DROP` means.